sched/task/task_spawn.c

```diff
@@ -131,12 +219,13 @@ static int nxtask_spawn_exec(FAR pid_t *pidp, FAR const char *name,
 
   /* Start the task */
 
-  pid = nxtask_create(name, priority, stackaddr,
-                      stacksize, entry, argv, envp);
+  pid = nxtask_spawn_create(name, priority, stackaddr,
+                            stacksize, entry, argv,
+                            envp ? envp : environ, actions);
```

Comment:
If envp = NULL is given from a higher layer, is it correct to grab the parent env here ?

Reply:
Yes, it is correct to inherits the parent's env directly

Reply:
Ok, thanks